### ISSUE TITLE
create and edit cluster policy / accept real dict for definition instead of pure string

### ISSUE THREAD

**Issue by copdips** [databricks/databricks-cli#554]
The API needs a pure string with espace on quotes for the value of the `definition` key:
https://docs.databricks.com/dev-tools/api/latest/policies.html#example

![image](https://user-images.githubusercontent.com/13083856/187650828-19445b24-f302-43fb-ae78-ed170c29b37a.png)

it would be nice in `databricks cluster-policies [create|edit]`, in the given json file, if the value of the `definition` key is a dict, then `databricks-cli` converts it to a pure string directly with escape. because adding quotes escape is not easy, and especially when saving the code in git, a definition in pure string is very unreadable,

hereunder, an example of pure dict policy, but we need to pre-convert the definition part to string.

```json
{
  "policy_id": "foo",
  "name": "bar",
  "definition": {
    "cluster_type": {
      "type": "fixed",
      "value": "job"
    }
}
```

**Comment by pietern** [comment]
@copdips That sounds reasonable. Would you want to submit a PR for this?

**Comment by copdips** [comment]
ok by this weekend

**Comment by pietern** [comment]
@copdips That's great, thank you. And there's no rush, so whatever is convenient.

Happy to work with you on getting this feature in.

**Comment by copdips** [comment]
resolved by #557
